debian/lcui-dev.install

```diff
@@ -1,4 +1,4 @@
 usr/include/*
-usr/lib/*/lib*.so
+usr/lib/*/lib*.so*
```

Comment:
`lcui_1.3.0-1_amd64.deb` file size is 1.9KB

![image](https://user-images.githubusercontent.com/1730073/70417172-a775f500-1a9b-11ea-9be9-95671988a762.png)

Reply:
Shouldn't you modify the `debian/lcui1.install` file?

Reply:
oops, absolutly